src/features/view-nft/HistoryItem/HistoryItem.tsx

```diff
@@ -71,7 +71,8 @@ export const HistoryItem = ({ item, tokenName }: HistoryItemProps) => {
 						secondaryAddress={item.seller}
 						amount={formatEthers(item.amount!) + tokenString}
 					/>
-					<Date timestamp={item.timestamp!} />
+					{/* NOTE: multiplying by 1000 here as transfer timestamps are in seconds for some reason */}
+					<Date timestamp={(Number(item.timestamp!) * 1000).toString()} />
```

Comment:
Random bug I spotted. Without the `*1000` it was displaying that events took place 10's of years ago